.cirrus.yml

```diff
@@ -142,39 +145,42 @@ task:
         # This uses --run-on-dirty-packages rather than --packages-for-branch
         # since only the packages changed by 'make-deps-path-based' need to be
         # checked.
-        - dart $PLUGIN_TOOL analyze --run-on-dirty-packages --log-timing --custom-analysis=script/configs/custom_analysis.yaml
+        - $PLUGIN_TOOL_COMMAND analyze --run-on-dirty-packages --log-timing --custom-analysis=script/configs/custom_analysis.yaml
         # Restore the tree to a clean state, to avoid accidental issues if
         # other script steps are added to this task.
         - git checkout .
-    # Does a sanity check that plugins at least pass analysis on the N-1 and N-2
-    # versions of Flutter stable if the plugin claims to support that version.
+    # Does a sanity check that packages at least pass analysis on the N-1 and N-2
+    # versions of Flutter stable if the package claims to support that version.
```

Comment:
nit: Should these still say `plugins`/`plugin`?

Reply:
All plugins are packages, so it's still correct as `packages`. I changed it because it makes diffing the files for actual differences easier, and I don't think we really lose any clarity by saying packages.